### PR TITLE
Test/send tx myself

### PR DESCRIPTION
**Pull request by alaibe**
Initial test for sending transaction
I miss the verify cause I need to fix the notification

statusq: https://github.com/status-im/StatusQ/pull/795

### CODE REVIEW

File: test/ui-test/testSuites/suite_status/tst_transaction/test.feature [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+    As a user I want to perform transaction
+
+    Background: Sign up & Enable wallet section & Toggle test networks
+		Given A first time user lands on the status desktop and generates new key
+		When user signs up with username tester123 and password TesTEr16843/!@00
+		Then the user lands on the signed in app
+		When the user opens app settings screen
+		When the user activates wallet and opens the wallet settings
+		When the user toggle test networks
```
Comment (jrainville) [inline test/ui-test/testSuites/suite_status/tst_transaction/test.feature:11]:
```suggestion
		When the user toggles test networks
```

**Review by jrainville — COMMENTED** [review]
Nice work, just some improvements on names

File: test/ui-test/src/screens/StatusWalletScreen.py [hunk 2]
```diff
@@ -121,3 +131,45 @@ def generate_new_account(self, account_name: str, password: str):
     def verify_account_name_is_present(self, account_name: str):
         verify_text_matching(MainWalletScreen.ACCOUNT_NAME.value, account_name)
         
+    def send_transaction(self, account_name, amount, token, chain_name, password):
+        click_obj_by_name(MainWalletScreen.SEND_BUTTON_FOOTER.value)
+        
+        accounts = get_obj(SendPopup.HEADER_ACCOUNTS_LIST.value)
```
Comment (noeliaSD) [inline test/ui-test/src/screens/StatusWalletScreen.py:137]: Maybe we can create a new `find_account` method and reuse the loop in next steps.

File: test/ui-test/testSuites/suite_status/shared/scripts/names.py [hunk 3]
```diff
@@ -148,15 +148,27 @@
...
+mainWallet_Send_Popup_Amount_Input = {"container": statusDesktop_mainWindow, "objectName": "amountInput", "type": "TextEdit"}
+mainWallet_Send_Popup_My_Accounts_Tab = {"container": statusDesktop_mainWindow, "objectName": "myAccountsTab", "type": "StatusTabButton"}
+mainWallet_Send_Popup_My_Accounts_List = {"container": statusDesktop_mainWindow, "objectName": "myAccountsList", "type": "StatusListView"}
+mainWallet_Send_Popup_Header_Accounts = {"container": statusDesktop_mainWindow, "objectName": "accountsListFloatingHeader", "type": "Repeater"}
+mainWallet_Send_Popup_Networks_List = {"container": statusDesktop_mainWindow, "objectName": "networksList", "type": "Repeater"}
+mainWallet_Send_Popup_Send_Button = {"container": statusDesktop_mainWindow, "objectName": "sendButtonPopup", "type": "StatusFlatButton"}
+mainWallet_Send_Popup_Password_Input = {"container": statusDesktop_mainWindow, "objectName": "passwordInput", "type": "StyledTextField"}
+
+
+mainWallet_Add_Account = {"container": statusDesktop_mainWindow, "text": "Add account", "type": "StatusBaseText", "unnamed": 1, "visible": True}
```
Comment (noeliaSD) [inline test/ui-test/testSuites/suite_status/shared/scripts/names.py:171]: Could we remove `text` property here?

File: test/ui-test/testSuites/suite_status/shared/steps/settingsSteps.py [hunk 4]
```diff
@@ -23,6 +23,10 @@ def step(context: any, account_name: str):
     _statusMain.open_settings()
     _settingsScreen.delete_account(account_name)
     
+@When("the user toggle test networks")
```
Comment (noeliaSD) [inline test/ui-test/testSuites/suite_status/shared/steps/settingsSteps.py:26]:
```suggestion
@When("the user toggles test networks")
```

File: test/ui-test/testSuites/suite_status/shared/steps/walletSteps.py [hunk 5]
```diff
@@ -1,7 +1,12 @@
 from screens.StatusWalletScreen import StatusWalletScreen
 
+_statusMain = StatusMainScreen()
 _walletScreen = StatusWalletScreen()
 
+@When("the user opens wallet screen")
+def step(context):
+    _statusMain.open_wallet()
+    
 @When("the user accept the signing phrase")
```
Comment (noeliaSD) [inline test/ui-test/testSuites/suite_status/shared/steps/walletSteps.py:10]:
```suggestion
@When("the user accepts the signing phrase")
```

;)

**Review by noeliaSD — APPROVED** [review]
LGTM! 
Minor comments added!

File: test/ui-test/src/screens/StatusWalletScreen.py (hunk 2 above)
Comment (alaibe) [reply test/ui-test/src/screens/StatusWalletScreen.py:137]: I did a click_repeater function

File: ui/imports/shared/views/SendModalHeader.qml [hunk 6]
```diff
@@ -40,6 +42,7 @@ StatusFloatingButtonsSelector {
             implicitHeight: 32
             defaultLeftPadding: 4
             text: name
+            objectName: name
```
[the remark this answers is not in the document]
Comment (alaibe) [reply ui/imports/shared/views/SendModalHeader.qml:45]: Not having the prefix makes it easer to map with the test, I want to click account "One" in multiple place, this allows to only say click on One, without the needs of the knowledge of where it is located. Plus as those are in list, what matters is the wrapper: list view/repeater having a more clear object name
WDYT?

File: test/ui-test/testSuites/suite_status/shared/scripts/names.py (hunk 3 above)
Comment (jrainville) [reply test/ui-test/testSuites/suite_status/shared/scripts/names.py:171]: I agree

**Review by jrainville — APPROVED** [review]
Looks good, I'd just remove the `text` from names.py if you can

**Pull request merged**